### PR TITLE
Document minimal billing unit

### PR DESCRIPTION
**Pull request by sammachin**
Update billing docs to reflect per second billing and suspending projects

### CODE REVIEW

File: docs/cloud/billing.md [hunk 1]
```diff
@@ -17,19 +17,19 @@ If a project is added during the billing cycle then a new charge is made for a f
 
 ## Removing Projects
 
-If you delete a project but still have other projects on their account you will receive a pro-rated credit for the number of whole days remaining in the billing cycle, which is used against a future invoice.
-If you delete your final project you will receive a pro-rated refund for the number of days remaining.
-The minimum cost for a project that is created and then deleted will be one day.
+If you delete a project but still have other projects on their account you will receive a pro-rated credit for the number of seconds remaining in the billing cycle, which is used against a future invoice.
```
Comment (ZJvandeWeg) [inline docs/cloud/billing.md:20]:
```suggestion
When a project is deleted you will receive a pro-rated credit for the number of seconds remaining in the billing cycle, which is used against a future invoice.
```

File: docs/cloud/billing.md [hunk 2]
```diff
@@ -17,19 +17,19 @@ If a project is added during the billing cycle then a new charge is made for a f
...
 
-## Stopped Projects
 
-Currently stopped projects are still charged as normal because the container is still running. 
-We hope to offer the ability to suspend a project in a future release, please see
-[the GitHub issue](https://github.com/flowforge/flowforge/issues/377).
+## Suspended Projects
+
+You are not billed for suspended projects.
+When you suspend a project your account will receive a credit for the number of seconds remaining in the billing cycle.
```
Comment (ZJvandeWeg) [inline docs/cloud/billing.md:26]:
```suggestion
When a project is suspended your account will receive a credit for the number of seconds remaining in the billing cycle.
```

File: docs/cloud/billing.md [hunk 3]
```diff
@@ -17,19 +17,19 @@ If a project is added during the billing cycle then a new charge is made for a f
...
+If you delete a project but still have other projects on their account you will receive a pro-rated credit for the number of seconds remaining in the billing cycle, which is used against a future invoice.
 
-## Stopped Projects
 
-Currently stopped projects are still charged as normal because the container is still running. 
-We hope to offer the ability to suspend a project in a future release, please see
-[the GitHub issue](https://github.com/flowforge/flowforge/issues/377).
+## Suspended Projects
+
+You are not billed for suspended projects.
```
Comment (ZJvandeWeg) [inline docs/cloud/billing.md:25]:
```suggestion
Suspended projects have no running editor, nor a runtime. You are not charged for suspended projects.
```

File: docs/cloud/billing.md (hunk 1 above)
Comment (ZJvandeWeg) [reply docs/cloud/billing.md:20]: I got confused by the if statement, what happens if the team is empty?

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]
Some minor nits.

**Comment by ZJvandeWeg** [comment]
Thanks for the quick turn around. Feel free to self merge when the requested changes are handled.

File: docs/cloud/billing.md (hunk 1 above)
Comment (sammachin) [reply docs/cloud/billing.md:20]: there's currently no difference if its the last project on the team or there are others, the account gets a credit and it sits on the account until there's an invoice.
We can manually  refund credit if they close their account and request it

**Pull request merged**